client/header/activity-panel/panels/inbox.js

```diff
@@ -33,6 +33,18 @@ class InboxPanel extends Component {
 		this.props.updateCurrentUserData( userDataFields );
 	}
 
+	handleActionClick( event, note_id, action_id ) {
+		const { triggerNoteAction } = this.props;
+		const href = event.target.href || '';
+
+		if ( 'http' === href.substr( 0, 4 ).toLowerCase() ) {
```

Comment:
@rrennick what do you think about using [`startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) here? It looks like a good case to use that method:
```suggestion
		if ( href.toLowerCase().startsWith( 'http' ) ) {
```